src/server/routes/competition.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const { createCompetition, getCompetitionById, } = require('../controllers/competition');
+const passport = require('passport');
+
+const router = express.Router();
+
+router.post('/', passport.authenticate('jwt', { session: false }), createCompetition);
```

Comment:
Oooo - this is interesting - I take it `passport` does some JWT auth middleware something or other (by the looks of it), then calls `next` to fire the function? Whatevs - nice work...